### ISSUE TITLE
CreationTime for cloudformation stacks is always a fixed value

### ISSUE THREAD

**Issue by n9iels** [localstack/localstack#2099]
## Issue description
When creating a CloudFormation stack, the `CreationDate` is always `2011-05-23T15:47:44Z`

## Steps to reproduce
1. Prepare a CloudFormation file, for example
```yml
AWSTemplateFormatVersion: 2010-09-09
Description: My Stack
Resources:
  LambdaCodeBucket:
    Type: AWS::S3::Bucket
    Properties:
      BucketName: my-bucket
```
2. Create a new stack based on this file, I used the command:
```bash
awslocal cloudformation create-stack --stack-name gch --template-body file://stack.yml
```
3. List all the created stacks and confirm the `CreationDate` is not a present date
```
awslocal cloudformation list-stacks
```
```
{
    "StackSummaries": [
        {
            "StackId": "arn:aws:cloudformation:us-east-1:000000000000:stack/stack/27187f13-a645-40a0-8788-62d502358216",
            "StackName": "stack",
            "TemplateDescription": "My Stack",
            "CreationTime": "2011-05-23T15:47:44Z",
            "StackStatus": "CREATE_COMPLETE"
        }
    ]
}
```

**Comment by whummer** [comment]
Thanks for reporting @n9iels . This should be fixed in #2103 , can you please give it a try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks

**Comment by n9iels** [comment]
The issue is fixed and the `CreationDate` is now correct, thanks!
